app/example_test.go

```diff
@@ -28,7 +28,6 @@ func Example() {
 	}
 
 	fmt.Printf("0x%x %s\n", node.ID(), node.Address())
-	// Output: 0x2dc171858c3155be 127.0.0.1:9001
```

Comment:
Why was this change needed? Has format of testable examples changed in Go 1.19? I would be very surprised.

Reply:
go 1.19 only allows a single Output comment block at the end of a function, and there is already one at the end, see e.g. [this](https://github.com/canonical/go-dqlite/actions/runs/4341772301/jobs/7581855576) test run.

```
# github.com/canonical/go-dqlite/app_test
Error: app/example_test.go:31:2: there can only be one output comment block per example
Error: app/example_test.go:95:2: output comment block must be the last comment block
```